docs/current_docs/api/custom-functions.mdx

```diff
@@ -63,6 +63,16 @@ Update the `src/MyModule.php` file with the following code:
 ```php file=./snippets/functions/functions-complex/php/src/MyModule.php
 ```
 </TabItem>
+<TabItem value="Java">
+
+Update the `src/main/java/io/dagger/modules/mymodule/MyModule.java` file with the following code:
+
+```java file=./snippets/functions/functions-complex/java/src/MyModule.php
```

Comment:
CI is failing as this path is incorrect: `ENOENT: no such file or directory, open '/opt/build/repo/docs/current_docs/api/snippets/functions/functions-complex/java/src/MyModule.php'`

Reply:
Thanks, it should be fixed now